src/qtscriptfuncs.cpp

```diff
@@ -33,18 +33,23 @@
 #include <QtCore/QJsonArray>
 #include <QtGui/QStandardItemModel>
 
+#include <time.h>
+#include <sys/time.h>
```

Comment:
This include (`#include <sys/time.h>`) is a problem for Windows builds.

The codebase isn't at all consistent on this (having originally been C), but it might be best to use `#include <ctime>` instead of both of these lines.

Reply:
I'll move this to loadsave.cpp and rewrite without `gettimeofday()`.